### PR TITLE
Add `Beatmap Spotlight Curators` article

### PR DESCRIPTION
**Pull request by Blooshing**
Added the BSC folder and the supplementary information that goes along with the usergroup. Much was taken from the already updated Spotlights page. As well as added it to the index and in `The Team` page as well

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Organisation
+
+The Beatmap Spotlights project is run by various community members across all game modes, led on by a dedicated project lead.
+
+| Role | Members |
+| :-- | :-- |
+| Project lead | ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) |
+| Project manager | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) |
+| Website manager | ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631), ::{ flag=US }:: [Snowleopard](https://osu.ppy.sh/users/3790227) |
```
Comment (Hiviexd) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:28]:
```suggestion
```
could be nuked, we haven't used the website for around 2 years nor intend to

**Review by Hiviexd — CHANGES_REQUESTED** [review]

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The Beatmap Spotlight Curators was formed as a dedicated community who assist in selecting high-quality maps for a given season of the [Spotlights](/wiki/Beatmap_Spotlights).
+
+## Becoming a curator
+
+In most cases, new Curators are suggested by the respective game mode team to fit a certain niche. Such suggestions are evaluated by the project managers, team leaders, and Curators themselves.
+
+In addition, there may be application cycles for certain game modes announced for certain seasons.
+
+All applicants are reviewed by project management and team leaders, and assessed based on their proficiency as a player, mapper, modder, and tournament mappool selector for their respective game mode. Applicants do not need to be proficient in all of these things to be selected, but having a broad skill set certainly helps. The roster of curators is based on a healthy mix and match of various skill levels, experiences, and proficiencies. The number of curators is being purposefully kept small and limited at the moment as the project develops.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:18]: for increased terseness and less repetition

```suggestion
All applicants are assessed based on their proficiency as a player, mapper, modder, and tournament mappool selector for their respective game mode. Applicants do not need to be proficient in all of these things to be selected, but having broad experience helps. The goal is to have a roster of curators with a mix of various skill levels, experiences, and proficiencies. However, the number of curators is currently kept small and limited while the project is developing.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+The Beatmap Spotlights project is run by various community members across all game modes, led on by a dedicated project lead.
+
+| Role | Members |
+| :-- | :-- |
+| Project lead | ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) |
+| Project manager | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) |
+
+## Curators
+
+Each Beatmap Spotlights release is assembled by a dedicated curator team and individually picked based on an extensive application and review process. Bold members are the respective team leaders.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:31]: curator application review was the topic of the previous section

```suggestion
Each Beatmap Spotlights release is assembled by a dedicated curator team. Team leaders are marked in bold.
```

File: wiki/People/The_Team/en.md [hunk 4]
```diff
@@ -46,6 +46,7 @@ The following are user groups which are made up of the osu! community members th
 | Name | Description |
 | :-- | :-- |
 | [Beatmap Nominators](Beatmap_Nominators) | Users going above and beyond the call to ensure your maps get qualified |
+| [Beatmap Spotlight Curators](Beatmap_Spotlight_Curators) | Responsible for selecting high-quality maps for seasonal Spotlights |
```
Comment (Walavouchey) [inline wiki/People/The_Team/en.md:49]:
```suggestion
| [Beatmap Spotlight Curators](Beatmap_Spotlight_Curators) | Responsible for selecting high-quality maps for the [Beatmap Spotlights](/wiki/Beatmap_Spotlights) |
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+---
+tags:
+    - BSC
+    - Beatmap Spotlight Curators
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:4]:
```suggestion
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
unsure about duplicating the lists of people, maybe remove those sections from the Beatmap Spotlights article, mention there that it is run by the BSC and link to here

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Becoming a curator
+
+In most cases, new curators are suggested by the respective game mode team to fit a certain niche. Such suggestions are evaluated by the project managers, team leaders, and curators themselves.
+
+In addition, there may be application cycles for certain game modes announced for certain seasons.
+
+All applicants are assessed based on their proficiency as a player, mapper, modder, and tournament mappool selector for their respective game mode. Applicants do not need to be proficient in all of these things to be selected, but having broad experience helps. The goal is to have a roster of curators with a mix of various skill levels, experiences, and proficiencies. However, the number of curators is currently kept small and limited while the project is developing.
+
+The [Beatmap Spotlight Curator(s) group page](https://osu.ppy.sh/groups/48) lists all of the team members. A more detailed member listing for all of the various roles within the Beatmap Spotlight Curators can be found on the respective [Beatmap Spotlight page](/wiki/Beatmap_Spotlights).
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:19]: delete

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+---
+tags:
+    - BSC
+    - beatmap spotlight curator
+---
+
+# Beatmap Spotlight Curators
+
+The **Beatmap Spotlight Curators** is a group that assists in selecting high-quality maps for each season of the [Beatmap Spotlights](/wiki/Beatmap_Spotlights).
```
Comment (Walavouchey) [inline wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:9]:
```suggestion
The **Beatmap Spotlight Curators** (***BSC***) is a group that assists in selecting high-quality maps for each season of the [Beatmap Spotlights](/wiki/Beatmap_Spotlights). The [Beatmap Spotlight Curators group page](https://osu.ppy.sh/groups/48) lists all of its members.
```

File: wiki/Beatmap_Spotlights/en.md [hunk 8]
```diff
@@ -13,6 +13,8 @@ The **Beatmap Spotlights** (or just *Spotlights*, formerly *Ranking Charts*) is
 
 The current season is the Summer 2022 season.
 
+The [Beatmap Spotlight Curator(s) group page](https://osu.ppy.sh/groups/48) lists all of the team members. A more detailed member listing for all of the various roles within the Beatmap Spotlight Curators can be found on the respective [Beatmap Spotlight Curators page](/wiki/Beatmap_Spotlights_Curators).
```
Comment (Walavouchey) [inline wiki/Beatmap_Spotlights/en.md:16]:
```suggestion
The [Beatmap Spotlight Curators group page](https://osu.ppy.sh/groups/48) lists all of the team members. See the [Beatmap Spotlight Curators article](/wiki/Beatmap_Spotlights_Curators) page for a more detailed listing of all the roles within the BSC.
```

File: wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md (hunk 6 above)
Comment (Blooshing) [reply wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:19]: Just to make sure that this is in reference to the entire `Becoming a Curator`, portion with the section about the group page, or is there a more specific portion that you are referencing to delete. Imo the comment is not as specific owo

Comment (Walavouchey) [reply wiki/People/The_Team/Beatmap_Spotlight_Curators/en.md:19]: it's kinda in the wrong section. at the listing or intro fits better

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**